examples/config-inference.yaml

```diff
@@ -11,7 +11,7 @@ logging:
   entity: your_wandb_entity
 validation_data:
   batch_size: 1
-  data_path: /path/to/validation/ic_0011
+  data_path: /path/to/data/directory
```

Comment:
question: Do we want to document what we expect to be present in this directory? e.g. netcdf files named in a particular way.

Reply:
I can add a short comment, but mostly I expect users to infer this from the example data we provide.